### DISCUSSION TITLE
Do you self host Raven or use it on Frappe Cloud?

### DISCUSSION THREAD

**Discussion by nikkothari22** [The-Commit-Company/raven#602] [General]
Asking this question to better understand where most of our users are right now. A lot of the work ahead (Web Push Notifications, a secret feature named "Three eyed Raven" 👀) depends on infrastructure.

**Comment by yahyaoudra** [comment]
self-hosted with erpnext

**Reply by dawoodjee** [reply]
You'll mostly get replies from selfhosteds, since they're the most folks with business coming here
